storm-core/src/ui/public/component.html

```diff
@@ -116,7 +116,7 @@
 
         var componentSummary = $("#component-summary");
         var componentActions = $("#component-actions");
-        var buttonJsonData = componentActionJson(response["encodedTopologyId"], response["encodedId"], response["id"],
+        var buttonJsonData = componentActionJson(response["name"], response["encodedId"], response["id"],
```

Comment:
We need to encode the name just like we conded the topologyId, or we have the possibility of an incorrect URL for odd characters in a topology name.